magento/magento2
# set correct remaining available discount amount when applying fixed p…

…rice cart rules to bundle products





### Description (*)


### Related Pull Requests


### Fixed Issues (if relevant)

1.  fixes https://github.com/magento/magento2/issues/30952

### Manual testing scenarios (*)

1. Install Magento 2.4 with sample data
2. Create cart rule with fixed discount amount for whole cart (Ex: 50$)
3. Add a bundle product to cart (Example: http://local.magento2.com/sprite-yoga-companion-kit.html)
4. Navigate to checkout/cart page and check discount.
5. Expected result: 50$, actual: 18.85$


### Questions or comments


### Contribution checklist (*)
